src/main/java/service/Report.java

```diff
@@ -0,0 +1,5 @@
+package service;
+
+public interface Report extends WritableToCsvFile {
```

Comment:
ReportService should be independent
```suggestion
public interface ReportService {
```